paddle/framework/ddim.cc

```diff
@@ -278,5 +278,11 @@ std::ostream& operator<<(std::ostream& os, const DDim& ddim) {
   return os;
 }
 
+ssize_t DDim::size() const { return product(*this); }
```

Comment:
Also, `size()` should be a member function for DDim. Invoke `product` is pretty strange to get the size.